### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by PeteUgwu**
## Views associated with models and controllers

In this Milestone, I:

- Implemented the design from the sneak peek wireframes
- Added minimal stylings
- Used methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,43 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# Create some users
+User.create!([
```
Comment (rustam-tolipov) [inline db/seeds.rb:10]: **Kudos for adding preloaded data to your project!** 🏆

File: config/routes.rb [hunk 2]
```diff
@@ -5,5 +5,5 @@
   # root "articles#index"
```
Comment (rustam-tolipov) [inline config/routes.rb]: [OPTIONAL] It would be great idea if you make users#index as a root

**Review by rustam-tolipov — APPROVED** [review]
Hi @PeteUgwu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
Kindly check comment below :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 2 above)
Comment (PeteUgwu) [reply config/routes.rb]: Thanks. Updated

File: db/seeds.rb (hunk 1 above)
Comment (PeteUgwu) [reply db/seeds.rb:10]: Thanks for your feedback

**Pull request merged**